docs/providers.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Providers
+icon: book-sparkles
+---
+
+Connect pulls data from `Providers`. `Providers` utilize public REST APIs and/or websockets from either centralized or decentralized exchanges. Below is a list of all available `Providers` in Connect.
+
+# Centralized Exchange Providers
+### REST API
+- binance_api
```

Comment:
I think for docs we can just list them as "Binance, Bitstamp, Coinbase" instead of the internal names we use.

Maybe we can link to the exchanges or their docs too